arch/x86/soc/ia32/soc.c

```diff
@@ -17,3 +17,16 @@
 #include <uart.h>
 #include <device.h>
 #include <init.h>
+#include <mmustructs.h>
+
+#ifdef CONFIG_MMU
+
+#ifdef CONFIG_XIP
+MMU_BOOT_REGION(CONFIG_PHYS_LOAD_ADDR, CONFIG_ROM_SIZE*1024, 1234);
```

Comment:
what does 1234 parameter mean?
Need to fix the flags values for all these calls.
Should be in terms of mmustructs.h macros.